templates/SECURITY.md

```diff
@@ -12,14 +12,30 @@ This project adheres to the [Eclipse Foundation Vulnerability Reporting Policy](
 
 ## How To Report a Vulnerability
 
-If you think you have found a vulnerability in `<project>` you can report it using one of the following ways:
+If you think you have found a vulnerability in this repository, please report it to us through coordinated disclosure.
+
+**Please do not report security vulnerabilities through public GitHub issues, discussions, or pull requests.**
```

Comment:
```suggestion
**Please do not report security vulnerabilities through public issues, discussions, or change requests.**
```

Reply:
(trying to keep wording platform agnostic)

Reply:
It is often better to mention HOW to do things, than how not to do things. Then maybe:

Please report security vulnerabilities using confidential methods only, like confidential and private tickets, or dedicated mailing lists.

Reply:
while in a template we should be generic ofc, mentioning things that are not possible with the platform do not make sense imho. 

GitHub does not support confidential or private tickets so it should not be mentioned here as this would just confuse people.